### DISCUSSION TITLE
Few questions on Android

### DISCUSSION THREAD

**Discussion by Cariboost** [rupeshs/fastsdcpu#265] [Q&A]
Hello, 

First of all thanks a lot for this wonderful "app" which I believe is the only one truly working on my Android phone (Galaxy note 10 +, android 12)

I'm curious about several things related to Android usage of fastsdcpu. I would enjoy if there were more documentation on this. 
Examples of questions :
> Is fastsdcpu able to use NPU on smartphone ? I see only mentions of PC support

> Overall are all the UI settings all working for Android ? For instance I noticed token merging has zero impact on anything, maybe because it is not working on Android ?

> Is there any option to not save every image automatically ?

Thanks !

**Comment by rupeshs** [comment]
@Cariboost Thanks for using FastSD.

**Is fastsdcpu able to use NPU on smartphone ? I see only mentions of PC support**
NPU supported only on AI PC.

**Overall are all the UI settings all working for Android ? For instance I noticed token merging has zero impact on anything, maybe because it is not working on Android ?**
Web UI should work fine on Android

**Is there any option to not save every image automatically ?**
By default images are automatically saved to "results" folder inside fastsdcpu directory.

**Reply by Cariboost** [reply]
Thanks to you sir for developping it ! 
One additional comment sir : you said by default images are automatically saved. But is there a way to change this default setting and only save when we want it ?

Thanks again

**Reply by rupeshs** [reply]
@Cariboost welcome, thanks for sponsoring.
Currently no option to disable image saving.
